packages/wp-now/src/pool.ts

```diff
@@ -0,0 +1,295 @@
+type instance = any;
+
+type request = (instance: instance) => Promise<any>;
+
+export type poolOptions = {
```

Comment:
Let's capitalize all type names – that's the convention used everywhere in Playground, and I think also mostly in TypeScript (except for primitives).

```suggestion
export type PoolOptions = {
```

Reply:
@adamziel https://github.com/WordPress/playground-tools/pull/110/files?diff=unified&w=1#diff-95251f05603f485be0377cd57ce90fe740821ca840c6053a115cb48c8725341bR5